dash-pipeline/bmv2/dash_headers.p4

```diff
@@ -241,6 +241,7 @@ header flow_data_t {
     bit<32> version;
     dash_flow_action_t actions;
     dash_meter_class_t meter_class;
+    bit<32> idle_timeout;
```

Comment:
will be better to split this PR into 2 and only keep the flow table id in this PR.

Reply:
Done

Reply:
Another one: https://github.com/sonic-net/DASH/pull/670